tests/Oryx.Tests.Common/StorageAccountEnvVariableForDb.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Oryx.BuildScriptGenerator.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Oryx.Tests.Common
+{
+    public static class StorageAccountEnvVariableForDb
+    {
+        public static List<EnvironmentVariable> AddStorageAccountEnvironmentVariables(List<EnvironmentVariable> envVarList)
```

Comment:
Minor, but personally, I think this functionality is more in line with an extension method on the `ICollection<EnvironmentVariable>` object.
That way we do not require the list class specifically but can add the env vars to any type of env var array/list. To summarize changes for this:

1. `List<EnvironmentVariable>` -> `ICollection<EnvironmentVariable>`
2. Add `this` keyword before `List<EnvironmentVariable> envVarList` parameter (to make it an extension method)
3. (nit) rename `StorageAccountEnvVariableForDb` to `EnvironmentVariableListExtensions` (or something similar)
4. (nit) rename `AddStorageAccountEnvironmentVariables` to `AddTestStorageAccountEnvironmentVariables`
5. change references of `StorageAccountEnvVariableForDb.AddStorageAccountEnvironmentVariables(buildEnvVariableList);` to `buildEnvVariableList.AddTestStorageAccountEnvironmentVariables();`

Reply:
@pauld-msft  This is a very good suggestion. I have updated the code accordingly apart from one change mentioned in your 2nd comment.

> ` 2. Add this keyword before List<EnvironmentVariable> envVarList parameter (to make it an extension method)`

Here, in place of the `List<EnvironmentVariable>`, I think we can use `ICollection<EnvironmentVariable>` to make the extension method generic. isn't it?